lib/ts/wrappers/index.ts

```diff
@@ -0,0 +1,21 @@
+/* Copyright (c) 2021, VRAI Labs and/or its affiliates. All rights reserved.
```

Comment:
Will the user be using this class at all? If yes, we need a proper way to expose this class. If not, then we don't need this file at all right?

Reply:
No, user will not use the file but we do need this for internal purpose. For e.g. in createNewSession:
```ts
import frameworks from "../../frameworks";
...
res = frameworks[SuperTokens.getInstanceOrThrowError().framework].wrapResponse(res);
```

Reply:
understood.